<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>signal_selection</string>
  </property>
  <widget class="QLineEdit" name="upper_range">
   <property name="geometry">
    <rect>
     <x>448</x>
     <y>110</y>
     <width>52</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QComboBox" name="condition_combox">
   <property name="geometry">
    <rect>
     <x>325</x>
     <y>110</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="lower_range">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>110</y>
     <width>52</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QComboBox" name="add_condition_combox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>230</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>230</width>
     <height>10</height>
    </size>
   </property>
   <property name="currentText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>437</x>
     <y>110</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>512</width>
     <height>192</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QListWidget" name="all_signals">
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
   </widget>
   <widget class="QListWidget" name="selected_signals"/>
  </widget>
  <widget class="QPushButton" name="file_open_button">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>50</y>
     <width>72</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>打开MDF</string>
   </property>
  </widget>
  <widget class="QPushButton" name="export_csv_button">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>380</y>
     <width>72</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>导出csv</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
